src/main/kotlin/objects/OBSClient.kt

```diff
@@ -202,6 +205,20 @@ object OBSClient {
             )
         }
 
+        try {
+            controller!!.registerTransitionListChangedCallback {
```

Comment:
Did you know (probably not because it isn't documented anywhere) that plugins can use ```OBSClient.preregisterCallback(callbackId: String, callback: (controller: OBSRemoteController) -> Unit)``` to register OBS callbacks? In that way plugins can use callbacks without needing a OBS Scene Queue code change. (```callbackId``` is just a random string, usually the plugin name + the callback name). 

The disadvantage of the obs-websocket-java dependency is that callbacks can only be registered once. That means that you won't be able to share the ```registerConnectCallback```. So, for the 'transition loading on connect' to work, I can suggest the following: 
1. The PR remains as is.
2. The ```loadTransistions()``` in the ```registerConnectCallback``` function moves to a ```refreshScenes``` or ```refreshOBSStatus``` method in a registered ```Refreshable``` class inside your plugin. 

The latter option (in combination with ```OBSClient.preregisterCallback```) has the advantage of making this (and maybe future) PR unnecessary. This may accelerate your plugin development (and keeps OBS Scene Queue cleaner). (```OBSClientStatus``` probably still needs to be updated for the added status).

Reply:
Totally makes sense--as I mention in my other comment, this could work as well--I defer to you on the final call.

If I do implement this, I will probably write a callback for plugins to attach to the `registerConnectCallback` method so that we can avoid triggering other events within the `loadScenes` method (just to keep concerns separated).